### ISSUE TITLE
Error when opening repo dir

### ISSUE THREAD

**Issue by arronpitman** [ipfs-shipyard/ipfs-desktop#1693]
Please describe what you were doing when this error happened.

**Specifications**

- **OS**: darwin
- **IPFS Desktop Version**: 0.13.0
- **Electron Version**: 9.3.2
- **Chrome Version**: 83.0.4103.122

**Error**

```
TypeError: shell.openItem is not a function
    at click (/Applications/IPFS Desktop.app/Contents/Resources/app.asar/src/tray.js:145:32)
    at MenuItem.click (electron/js2c/browser_init.js:65:1624)
    at Function.executeCommand (electron/js2c/browser_init.js:73:1102)
```

**Comment by jessicaschilling** [comment]
Thanks @arronpitman - this is a known issue and we are expecting to ship a fix early next week! Thanks.

**Comment by lidel** [comment]
Closed in #1676
